test/util/externalservices/AuditService.scala

```diff
@@ -23,14 +23,22 @@ object AuditService {
 
   private val AuditWriteUrl: String = "/write/audit"
 
-  def stubAuditService(): Unit =
+  def stubAuditService(): Unit = {
     stubFor(
       post(urlEqualTo(AuditWriteUrl))
         .willReturn(
           aResponse()
             .withStatus(Status.OK)
         )
     )
+    stubFor(
```

Comment:
First stubbed url is one that our code calls explicitly.   I think the "/merged" calls are being done by the Auditing library and are resulting in a lot of "missed" matches in WireMock output.